frameworks/PHP/swoole/swoole-server.php

```diff
@@ -10,7 +10,7 @@
 
 $server  = new Server('0.0.0.0', 8080);
 $setting = [
-    'worker_num'        => swoole_cpu_num() * 4,
+    'worker_num'        => swoole_cpu_num(),
```

Comment:
This is correct for the async, or all coroutines will fight for the CPU.
But not for the sync, that use the same OS context switch and need more processes for db connections.

Reply:
I have already considered this situation now.